apps/ledger-live-desktop/src/renderer/families/ethereum/MaxBaseFeeField.jsx

```diff
@@ -0,0 +1,137 @@
+// @flow
+import React, { useCallback, useMemo } from "react";
+import { BigNumber } from "bignumber.js";
+import invariant from "invariant";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+import type { Account, TransactionStatus } from "@ledgerhq/live-common/types/index";
+import type { Transaction } from "@ledgerhq/live-common/families/ethereum/types";
+import { formatCurrencyUnit } from "@ledgerhq/live-common/currencies/index";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import { inferDynamicRange } from "@ledgerhq/live-common/range";
+import Label from "~/renderer/components/Label";
+import FeeSliderField from "~/renderer/components/FeeSliderField";
+import InputCurrency from "~/renderer/components/InputCurrency";
+import Box from "~/renderer/components/Box";
+import LabelWithExternalIcon from "~/renderer/components/LabelWithExternalIcon";
+import { openURL } from "~/renderer/linking";
+import { track } from "~/renderer/analytics/segment";
+import { urls } from "~/config/urls";
+import TranslatedError from "~/renderer/components/TranslatedError";
+
+const ErrorContainer = styled(Box)`
+  margin-top: 0px;
+  font-size: 10px;
+  width: 100%;
+  transition: all 0.4s ease-in-out;
+  will-change: max-height;
+  max-height: ${p => (p.hasError ? 60 : 0)}px;
+  min-height: ${p => (p.hasError ? 22 : 0)}px;
+`;
+  
+const ErrorDisplay = styled(Box)`
+  color: ${p => p.theme.colors.pearl};
+`;
+
+const WarningDisplay = styled(Box)`
+  color: ${p => p.theme.colors.warning};
+`;
+
+type Props = {
+  account: Account,
+  transaction: Transaction,
+  status: TransactionStatus,
+  updateTransaction: (updater: any) => void,
+};
+
+const FeesField = ({
+  account,
+  transaction,
+  status,
+  updateTransaction
+}: Props) => {
+  invariant(transaction.family === "ethereum", "FeeField: ethereum family expected");
```

Comment:
what happens if this is actually called ? Does a modal error appear or app crash ?